picard/tagger.py

```diff
@@ -367,6 +367,9 @@ def add_files(self, filenames, target=None):
             if ignore_hidden and is_hidden(filename):
                 log.debug("File ignored (hidden): %r" % (filename))
                 continue
+            if os.path.basename(filename).startswith(".smbdelete") and is_hidden(filename):
```

Comment:
Does not make sense to have just 1 extension as a special case. Can we instead create a new constant list in picard.constant, and check force all the files with the extension in that extension list?

Reply:
I agree in general - but can we leave it until we have a second case to create a general solution?

Reply:
Okay, can you add a comment explaining this special case then? And a TODO for future reference when we do need to shift to a general case

Reply:
Is there a specific need to test if the file is hidden, i think we want to ignore all files started with `.smbdelete` whether they are hidden or not.
For reference, this kind of files are created by http://www.opensource.apple.com/source/smb/smb-759.40.1/kernel/smbfs/smbfs_smb.c

About handling general case, no need for now, just add a comment about why we need to ignore such files.